distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1009,13 +1011,13 @@ public void testCatalogEndpointExposure() throws InvalidSyntaxException {
 
     @Test
     public void testCswSubscriptionByWildCardSearchPhrase() throws Exception {
-        whenHttp(server).match(Condition.post(SUBSCRIBER))
+        whenHttp(server).match(post(SUBSCRIBER))
```

Comment:
✏️  We should be consistent and always use `Condition` or do static imports for all of them.